modin/core/io/sql/sql_dispatcher.py

```diff
@@ -149,7 +149,9 @@ def write(cls, qc, **kwargs):
         # are not pickleable. We have to convert it to the URL string and connect from
         # each of the workers.
         if cls._is_supported_sqlalchemy_object(kwargs["con"]):
-            kwargs["con"] = str(kwargs["con"].engine.url)
+            kwargs["con"] = kwargs["con"].engine.url.render_as_string(
+                hide_password=False
```

Comment:
Is it safe to not hide the password?

Reply:
The alternative to using a SQLAlchemy connection object (just passing the connection string instead) doesn't hide the password.